src/mame/drivers/missb2.cpp

```diff
@@ -518,7 +508,7 @@ ROM_START( missb2 )
 	ROM_LOAD( "msbub2-u.203", 0x10000, 0x10000, CRC(29fd8afe) SHA1(94ead80d20cd3974dd4fb0358915e3bd8b793158) )
 	/* 20000-2ffff empty */
 
-	ROM_REGION( 0x10000, "slave", 0 ) /* 64k for the second CPU */
+	ROM_REGION( 0x10000, "subcpu", 0 ) /* 64k for the second CPU */
```

Comment:
Do we have missb2 schematics?  I'm sorry if we do.  I was more irritated about the `missb2_` on the name of a static address map in a file called `missb2.cpp` - there's no need to put a prefix to avoid collisions when it's static, and it's obvious it's for `missb2` given the name of the file.